docs/development/Serial.md

```diff
@@ -94,14 +94,39 @@ Note: for Identifier see serialPortIdentifier_e in the source; for Function bitm
 | SERIAL_PORT_SOFTSERIAL1 |    30 |
 | SERIAL_PORT_SOFTSERIAL2 |    31 |
 | SERIAL_PORT_LPUART1     |    40 |
+| SERIAL_PORT_UART0       |    50 |
+| SERIAL_PORT_USART1      |    51 |
+| SERIAL_PORT_USART2      |    52 |
+| SERIAL_PORT_USART3      |    53 |
+| SERIAL_PORT_UART4       |    54 |
+| SERIAL_PORT_UART5       |    55 |
+| SERIAL_PORT_USART6      |    56 |
+| SERIAL_PORT_USART7      |    57 |
+| SERIAL_PORT_USART8      |    58 |
+| SERIAL_PORT_UART9       |    59 |
+| SERIAL_PORT_USART10     |    60 |
+
+Firmware 4.6 changes the way CLI handles serial configuration as it uses serial port name instead of identifier - restoring previous diff won't be possible (breaking port configuration)
```

Comment:
It'll restore an indexed backup no problem... you won't be able to restore a 4.6 backup on earlier versions (eg 4.5).